test/Bart/Git/CommitTest.php

```diff
@@ -39,6 +56,15 @@ public function testMessage()
 		$this->assertContains('a57a2664feafb26c61d269babc63b272ed87544d', $commit->message(), 'hash');
 	}
 
+	public function testgetFileList()
```

Comment:
Rename to `getGetFileList()`

Please also test for the case when the command fails (i.e. `$result` was not OK).